tests/test_utils.py

```diff
@@ -637,29 +637,6 @@ def walk(s, recurse):
     }
 
 
-def test_handle_function_schema():
```

Comment:
This test was added for coverage purposes, because as mentioned the method couldn't be called in "real" code